pallets/committee/src/benchmarking.rs

```diff
@@ -0,0 +1,203 @@
+where
+    I: Instance,
+    T: Trait<I>,
+{
+    ensure!(
+        users.len() > 0,
+        "make_proposals_and_vote requires non-empty users"
+    );
+    for i in 0..PROPOSALS_NUM {
+        let index = Module::<T, I>::proposal_count();
+        let proposal = make_proposal::<T, I>(i + 1).0;
+        identity::CurrentDid::put(users[0].did());
+        Module::<T, I>::vote_or_propose(users[0].origin.clone().into(), true, Box::new(proposal))?;
+        if users.len() > 1 {
+            let hash = *Module::<T, I>::proposals()
+                .last()
+                .ok_or("missing last proposal")?;
+            // cast min(user.len(), N) - 1 additional votes for proposal #N
+            for (j, user) in users.iter().skip(1).take(i as usize).enumerate() {
+                // Vote for the proposal if it's not finalised.
+                if Module::<T, I>::voting(&hash).is_some() {
+                    identity::CurrentDid::put(user.did());
+                    Module::<T, I>::vote(user.origin.clone().into(), hash, index, j % 2 == 0)?;
+                }
+            }
+        }
+    }
+    Ok(())
+}
+
+fn make_members_and_proposals<T, I>() -> Result<Vec<User<T>>, DispatchError>
+where
+    I: Instance,
+    T: Trait<I>,
+{
+    let members: Vec<_> = (0..COMMITTEE_MEMBERS_MAX)
+        .map(|i| user::<T>("member", i))
+        .collect();
+    Members::<I>::put(members.iter().map(|m| m.did()).collect::<Vec<_>>());
+    make_proposals_and_vote::<T, I>(&members)?;
+    Ok(members)
+}
+
+benchmarks_instance! {
+    _ {}
+
+    set_vote_threshold {
+        let n = 1;
+        let d = 2;
+        let origin = T::CommitteeOrigin::successful_origin();
+        let call = Call::<T, I>::set_vote_threshold(n, d);
+    }: {
+        call.dispatch_bypass_filter(origin)?;
+    }
+    verify {
+        ensure!(Module::<T, _>::vote_threshold() == (n, d), "incorrect vote threshold");
+    }
+
+    set_release_coordinator {
+        let dids: Vec<_> = (0..COMMITTEE_MEMBERS_NUM).map(|i| user::<T>("member", i).did()).collect();
+        let coordinator = dids.last().unwrap().clone();
+        Members::<I>::put(dids);
+        let origin = T::CommitteeOrigin::successful_origin();
+        let call = Call::<T, I>::set_release_coordinator(coordinator);
+    }: {
+        call.dispatch_bypass_filter(origin)?;
+    }
+    verify {
+        ensure!(
+            Module::<T, _>::release_coordinator() == Some(coordinator),
+            "incorrect release coordinator"
+        );
+    }
+
+    set_expires_after {
+        let maybe_block = MaybeBlock::Some(1.into());
+        let origin = T::CommitteeOrigin::successful_origin();
+        let call = Call::<T, I>::set_expires_after(maybe_block);
+    }: {
+        call.dispatch_bypass_filter(origin)?;
+    }
+    verify {
+        ensure!(Module::<T, _>::expires_after() == maybe_block, "incorrect expiration");
+    }
+
+    vote_or_propose_new_proposal {
+        let members = make_members_and_proposals::<T, I>()?;
```

Comment:
They are. The constant is called `COMMITTEE_MEMBERS_MAX`.

Reply:
They are currently hardcoded at `COMMITTEE_MEMBERS_MAX`. My suggestion was to make it a variable so that we can see the effect of increasing the number of members.